__tests__/app.test.js

```diff
@@ -68,4 +69,45 @@ describe("app", () => {
         });
     });
   });
+
+  describe("GET /api/reviews/:review_id", () => {
+    test("200: responds with a specific review object, which should have 9 properties", () => {
+      return request(app)
+        .get("/api/reviews/10")
+        .expect(200)
+        .then((response) => {
+          const review = response.body;
+          expect(review).toMatchObject({
```

Comment:
We want to check the specific data here as we're trying to grab one specific review